openhtf/core/test_executor.py

```diff
@@ -109,7 +109,7 @@ def finalize(self):
   def wait(self):
     """Waits until death."""
     try:
-      self.join(sys.float_info.max)  # Timeout needed for SIGINT handling.
+      self.join(315400000)  # Timeout needed for SIGINT handling.
```

Comment:
Why was this changed? PY3.5 has this still

Reply:
I get this: OverflowError: timestamp too large to convert to C _PyTime_t

Reply:
sys.float_info.max isn't a very reasonable value to pass to self.join, even as a placeholder. I'd suggest replacing them with something like "60" and leaving a comment saying that the timeout is not actually used.

Reply:
Made update